Feature 23 better repositories

Alfro opened the pull request:
This feature adds a command with support for improved repositories metadata.
Code review.

Alfro committed:
#23 FeedSyncService: Added new syncRepos function.

This function will be used to add metadata to repositories tags based on a json file.

Alfro committed:
#23 Command: Created SyncReposMetadata Command to launch the syncRepos function.

Alfro committed:
#23 views/Catalogue: Tweaked thumbnail template to display new metadata

Alfro committed:
#23 Resources: Added some images and json

Added the image and json for CAMPUSDOMAR, the json for VIGO and a default picture for the repositories with no json currently.

Alfro committed:
#23 Resources: Improved description viewing. Added uVigoTV image.

rubenrua commented on Services/FeedSyncService.php:
```diff
@@ -375,4 +376,51 @@ protected function showProgressEstimateDuration($time_started, $processed, $tota
+            $progressBar->advance();
+        }
+        $progressBar->finish();
+        echo "\nALL LOADED\n";
```
`echo` inside a service????

rubenrua commented on Services/FeedSyncService.php:
```diff
@@ -375,4 +376,51 @@ protected function showProgressEstimateDuration($time_started, $processed, $tota
+    public function syncRepos($output, $optWall, $show_bar)
+    {
+
+        $dataFolder = $this->dataFolder->locateResource('@PumukitGeantWebTVBundle/Resources/data/repos_data');
```
We thought about searching the repo data on the github site, not in the source code, innit?

Alfro committed:
#23 Added option 'repos-directory' to command.

Alfro committed:
#23 Modified FeedSyncService to use parameter 'reposDir' if not null.

Alfro committed:
#23 Added git clone of the project to app/cache

This way we make sure to retrieve updated version of json files on data/repos_data

Alfro committed:
FeedSyncService: Changed echo for 'output'

Alfro committed:
#23 repos_data/CAMPUSDOMAR.json: Changed image name

Alfro commented:
Merging into 1.0.x first, not master.
